demos/dermatology.html

```diff
@@ -193,9 +193,14 @@
         var currWidget = null;
         var showProperties = null;
         var debouncedResize = null;
+        var propEditor = null;
 
         require(["src/layout/Surface", "src/layout/Grid", "src/other/Persist", "src/other/PropertyEditor"], function (Surface, Grid, Persist, PropertyEditor) {
-            var propEditor = new PropertyEditor().showColumns(false).showData(false);
+            propEditor = new PropertyEditor()
+                    .showColumns(false)
+                    .showData(false)
+                    .excludeTags(["Advanced","Private"])

```

Comment:
I think in dermatology we want to see all the properties by default - thats all?